src/shaders/textureVisualizer.frag

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+precision highp float;
+
+// ------------ input -----------------------
+in highp vec2 textureCoordinates;
+
+// ------------ uniforms --------------------
+uniform highp sampler2D sourceTexture;
+
+uniform highp sampler2D colorMapTexture;
+uniform highp vec2 colorMapOffsetScale;
+#define colorMapOffset colorMapOffsetScale.x
+#define colorMapScale colorMapOffsetScale.y
+
+#ifdef DEPTH_TEXTURE
+uniform highp vec2 depthUnprojection;
+#endif
+
+//------------- output ----------------------
+#ifdef EXPLICIT_ATTRIB_LOCATION
+layout(location = OUTPUT_ATTRIBUTE_LOCATION_COLOR)
+#endif
```

Comment:
I think this could do without the `#ifdef EXPLICIT_ATTRIB_LOCATION` around -- in Magnum that's only for compatibility with GL 2.1 / ES2 / WebGL 1, which you don't care about here.

(Also remove the related `addSource()` in the shader cpp, of course.)